gapis/service/service.proto

```diff
@@ -50,6 +50,7 @@ message Commands {
 }
 message Devices {
   repeated path.Device list = 1;
+  repeated IncompatibleDevice incompatibleList = 2;
```

Comment:
I'd prefer to just have a `repated device.Incompatability` field here, where the two lists are the same length and correspond to each other. This is a common proto practice. Otherwise just a repeated message with both fields. I.e. one of these:
```
message Devices {
  repeated path.Device list = 1;
  repeated device.Incompataiblity replayCompatability = 2;
}
```
or
```
message Device {
  path.Device device = 1;
  device.Incompataiblity replayCompatability = 2;
}

message Devices {
  repeated Device list = 1;
}
```

 Also please document here that this second field is only set for the `GetDevicesForReplay` call, and not `Get(Path.Devices)`.

Reply:
I changed to the first of the options you suggested:

```
message Devices {
  repeated path.Device list = 1;
  repeated device.Incompatibility replayCompatibilityList = 2;
}
```

Note that this impacts the semantics of GetDevicesForReplayResponse, where the device list now contains both compatible and incompatible devices.  They are sorted to make sure compatible devices always come first.